examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,202 @@
+<html lang="en">
+    <head>
+        <title>Itowns - Pick positions</title>
+
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+    <body>
+    	<div id="description">
+	        key bindings
+	        <ul>
+	        	<li>Left-Click: camera translation (drag)</li>
+		        <li>Right-Click: camera translation (pan)</li>
+    		    <li>Ctrl + Left-Click: camera rotation (orbit)</li>
+    		    <li>Mouse Wheel: zoom in/out</li>
```

Comment:
Could add 2 Titles to separate controls and recording.
S key is already used to select tile.


Could you use [THREE.CameraHelper](https://threejs.org/docs/#api/en/helpers/CameraHelper) to show recorded position?

Reply:
I added two separate titles for controls and recording.
I switched the `S` key for the `P` key.
I used `THREE.CameraHelper` to show recorded position.